### PR TITLE
Milestone # 1: Completed NYT Project

### PR DESCRIPTION
**Pull request by santiagorodriguezbermudez**
## Milestone 1:

In this milestone we accomplished:
- Created HTML structure for cloning the New York Times article
- Styled the HTML structure accordingly
- Corrected linters and HTML markup

### CODE REVIEW

**Comment by santiago-dev96** [comment]
## Pull Request under review by @santiago-rodrig (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+                        </div>
+                        <h3>A Nest of Alien Asteroids Orbits Our Sun
+                        </h3>
+                        <span class="posted-date">12 Aug 2020</span>
+                    </div>
+                </div>
+                <br>
+                <hr class="hr-picks">
+                <br>
+                <h3 class="content-header">Editors picks</h3>
```
Comment (santiago-dev96) [inline index.html]: You must use an aside

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required :no_entry_sign:

1. You must use an `aside` tag for the "Editors' picks" part

image

After implementing the changes submit another CRR (Code Review Request). You can find me in slack as **srodrig**, have a nice day :wave:!

File: index.html (hunk 1 above)
Comment (simandebvu) [reply index.html]: ## Code Review #1 
> Added aside tag @ recommended by @santiago-rodrig

**Comment by simandebvu** [comment]
> Added aside tag

**Comment by simandebvu** [comment]
## Code Review #1 

> Added aside tag as suggested by @santiago-rodrig

**Review by serdg0 — APPROVED** [review]
**Congratulations** @simandebvu :fireworks: :100: :fireworks: 

Feel free to merge.

Keep the good work in your next project!

**Happy Coding!** :computer: :nerd_face: 

**_Sergio Diaz_**

**Pull request merged**